debian/focal/foreman/foreman.install

```diff
@@ -7,6 +7,7 @@ script/foreman-rake                      usr/sbin
 script/foreman-tail                      usr/sbin
 debian/foreman-plugin.mk                 usr/share/foreman
 debian/foreman-plugin.postinst-lib       usr/share/foreman
+debian/foreman.postinst-lib              usr/share/foreman
```

Comment:
I don't see this change for bullseye. Am I missing something?

Reply:
You are not, I am ;)